meta-openpower/recipes-phosphor/host/op-proc-control_git.bb

```diff
@@ -11,17 +11,23 @@ inherit autotools obmc-phosphor-utils pkgconfig pythonnative
 inherit systemd
 
 SRC_URI += "git://github.com/openbmc/openpower-proc-control"
```

Comment:
Shouldn't this be pointing to our fork and OP940 branch? (i.e. https://github.com/ibm-openbmc/openpower-proc-control) ?

Reply:
Andrew , Can you please check now?